docs/performance-dashboard.md

```diff
@@ -0,0 +1,86 @@
+# Performance Dashboard
+
+## Overview
+
+Performance dashboards can be created in Sentry to visualise the evolution of a given performance metric over time.
+
+## Choose the trace you want to monitor
+
+The performance metric needed to monitor the performance of a given user flow can be tracked by adding a performance trace to the codebase (or by using an existing trace if one already exists). More details on performance tracing can be found [here](./performance-tracing.md).
+
+## Choose the dashboard you want to use
+
+Go to [Sentry dashboards page](https://metamask.sentry.io/dashboards/).
+
+From there, you can either create a new dashboard for you team, or update an existing one.
+
+If you create a new dasboard, you'll need to indicate the name of the project:
+- Choose `metasmask` project for Extension
+- Choose `metasmask-mobile` project for Mobile
+
+Example:
+- [Extension Platform team dashboard](https://metamask.sentry.io/dashboard/115782/)
+- [Mobile Platform team dashboard](https://metamask.sentry.io/dashboard/115346/)
+
+## Create a new widget
+
+Click on "Add widget".
+
+### 1. Name your widget
+
+Choose a self explanatory name for your widget, like: `name-of-the-trace (type-of-application)`.
+
+Example: `UI Startup (Chrome)` or `UI Startup (Firefox)` or `UI Startup (iOS)` or `UI Startup (Android)`
+
+### 2. Choose your visualization
```

Comment:
Suggestion: Can we add suggestions of other visualizations we can choose? eg: Area chart, Line Chart

Reply:
Fixed in this [commit](https://github.com/MetaMask/contributor-docs/pull/129/commits/0206101648b01007ce50010ce923ff775c54156b)